examples/exporting_for_inference.ipynb

```diff
@@ -7,22 +7,7 @@
    "source": [
     "# 🥡 Exporting for Inference\n",
     "\n",
-    "Composer models are also `torch.nn.Module`, and thus can be exported like any other PyTorch module. In this tutorial, we walk through how to export your models into various common formats: [ONNX](https://onnx.ai/), [TorchScript](https://pytorch.org/docs/stable/jit.html), and [torch.fx](https://pytorch.org/docs/stable/fx.html). For more detailed options and configuration settings, please consult the linked documentation.\n",
-    "\n",
-    "## Algorithm compatibility\n",
-    "Some of our algorithms alter the model architecture in ways that may render them incompatible with some of the export procedures above. For example, BlurPool replaces some instances of `Conv2d` with `BlurConv2d` layers which are not compatible with `torch.fx` as they have data-dependant control flow. \n",
```

Comment:
I think this table is important to include, for reference, correct?

Reply:
I moved it to the bottom. Our algorithm compatibility will improve so didn't want to start with highlighting what doesn't work.